scripts/update/radarr.sh

```diff
@@ -64,12 +64,22 @@ if [[ -f /install/.radarr.lock ]]; then
     #Mandatory SSL Port change for Readarr
     #shellcheck source=sources/functions/utils
     . /etc/swizzin/sources/functions/utils
-    [[ -z $radarrOwner ]] && radarrOwner=$(_get_master_username)
+    app_name="radarr"
+    if [ -z "$RADARR_OWNER" ]; then
```

Comment:
let's stick to the old `radarrOwner` convention for now so we can replace it all later in one go, and prevent more doc updates than necessary